mdpow/analysis/ensemble.py

```diff
@@ -384,15 +384,11 @@ def select_atoms(self, *args, **kwargs):
                 raise
             else:
                 selections[key] = ag
-        return EnsembleAtomGroup(selections, ensemble_dir=self._ens_dir)
+        return EnsembleAtomGroup(selections, ensemble=self._ensemble)
 
     def ensemble(self):
```

Comment:
Make `.ensemble` a managed property, similar to `.universe`.
```suggestion
    @property
    def ensemble(self):
```